app/assets/javascripts/index/query.js

```diff
@@ -221,16 +225,15 @@ OSM.Query = function (map) {
             .text(I18n.t("javascripts.query.nothing_found"))
             .appendTo($ul);
         }
-      },
-      error: function (xhr, status, error) {
+      })
+      .catch(error => {
         $section.find(".loader").hide();
 
         $("<li>")
           .addClass("list-group-item")
-          .text(I18n.t("javascripts.query." + status, { server: url, error: error }))
+          .text(I18n.t("javascripts.query.error", { server: url, error: error.message }))
           .appendTo($ul);
-      }
-    }));
+      });
```

Comment:
Missing controller cleanup?

Reply:
Added, though I don't believe it really matters much - it just means on the next query we'll send an abort on a controller that has nothing listening to it's signal.

Reply:
Thanks. 
Will now result in a bit to feed the garbage collector (potentially the whole fetch tree with data).